docs/hands-on-lab/05-implementing-the-transport-hints.md

```diff
@@ -0,0 +1,19 @@
+## Implementing the transport - Exercise
```

Comment:
@DavidS I'm confused, what is the difference between this section and 'Implementing the transport' below? The other one seems like it is the exercise.

Reply:
These are additional exercise steps that can be used to deepen the reader's understanding and comfort with the code. They're not necessary for completing the Lab.